torch/testing/_internal/common_nn.py

```diff
@@ -5044,6 +5044,10 @@ def fn_to_gradcheck(*inputs_and_params, **kwargs):
             assert not kwargs
             return test_case._forward(module, inputs_and_params[:num_inputs])
 
+        # These modules don't actually take the arguments as inputs so forward check will fail for these
+        # We will be able to re-enable this when https://github.com/pytorch/pytorch/issues/49171 is done
+        check_forward = len(params) == 0
```

Comment:
We should be more descriptive here and name it "check_forward_mode_ad". Otherwise, there is the ambiguity if we are talking about forward mode AD or the forward pass of the module

Reply:
That's a very good point. I forgot that there was anything else but forward mode AD :D